src/matchers/general/toMatchSnapshot.ts

```diff
@@ -0,0 +1,28 @@
+import { toMatchSnapshot as jestToMatchSnapshot, toMatchInlineSnapshot as jestToMatchInlineSnapshot } from 'jest-snapshot'
```

Comment:
We should define this as peer dependency

Reply:
noted with thanks!

Reply:
updated